dev/UndockedRegFreeWinRT/typeresolution.cpp

```diff
@@ -213,8 +221,19 @@ namespace UndockedRegFreeWinRT
         _COM_Outptr_opt_result_maybenull_ IMetaDataImport2** ppMetaDataImport,
         _Out_opt_ mdTypeDef* pmdTypeDef)
     {
+        if (ppMetaDataImport != nullptr)
+        {
+            *ppMetaDataImport = nullptr;
+        }
+        if (pmdTypeDef != nullptr)
+        {
+            *pmdTypeDef = 0;
+        }
+
         wchar_t szCandidateFileName[MAX_PATH + 1]{};
         HRESULT hr{ StringCchCopy(szCandidateFileName, ARRAYSIZE(szCandidateFileName), pszFullName) };
+        hr = WindowsCreateString(L"", 0, phstrMetaDataFilePath);
```

Comment:
Is it intentional that we now overwrite the hr result from the stringCchCopy() w/o first checking for error?

Reply:
oh good point